packages/adblocker-electron/adblocker.ts

```diff
@@ -161,7 +167,43 @@ export class ElectronBlocker extends FiltersEngine {
     event.returnValue = this.config.enableMutationObserver;
   };
 
-  public onGetCosmeticFilters = (
+  public onGetCosmeticFiltersFirst = (event: Electron.IpcMainEvent, url: string) => {
```

Comment:
Could we somehow keep a single function for `onGetCosmeticFiltersFirst` and `onGetCosmeticFiltersUpdated` to prevent duplication?

Reply:
Of course, feel free to refactor the code inside the PR if you see fit.

Regarding onGetCosmeticFilters, I thought it might be better to have a separate method for when the function is called initially with sendSync and needs to synchronously return, versus when the DOM is updated and it doesn't need to be synchronous. What do you think?

Reply:
Sounds good, it was mostly a nit comment. Let's go with the current solution.